tests/test_datetime_utils.py

```diff
@@ -1,11 +1,9 @@
 import unittest
 from datetime import datetime, timedelta
 
-from dateutil.utils import today
-
 from core_lib.helpers.datetime_utils import year_begin, year_end, month_begin, month_end, week_begin, week_end, \
-    day_begin, day_end, tomorrow, yesterday, midnight, sunday, monday, tuesday, wednesday, thursday, friday, saturday, \
-    hour_begin, hour_end
+    day_begin, day_end, tomorrow, today, yesterday, midnight, sunday, monday, tuesday, wednesday, thursday, friday, \
+    saturday, hour_begin, hour_end
 from core_lib.helpers.func_utils import reset_datetime
 
```

Comment:
where is the test for the `today` function?

Reply:
the test was already written but it had imported today() `from dateutil.utils import today` from here, so while adding hints i noticed that

Reply:
thank you. didn't follow. do we have a today tesT?

Reply:
please can we add test. to  assert `from dateutil.utils import today` with core-lib `today function`
thank you